entries/v/vaultcord.com.json

```
@@ -0,0 +1,15 @@
+{
+  "VaultCord": {
+    "domain": "vaultcord.com",
```

The additional-domains element helps link any other domains to the service.
```suggestion
    "domain": "vaultcord.com",
    "additional-domains": [
      "novanode.win"
    ],
```